packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/xdd/DocumentResource.java

```diff
@@ -0,0 +1,50 @@
+package software.uncharted.terarium.hmiserver.resources.xdd;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import software.uncharted.terarium.hmiserver.proxies.xdd.DocumentProxy;
+
+@Path("/api/xdd/documents")
```

Comment:
Need `@Authenticated` so that the API Service will confirm the bearer token is valid

Reply:
fixed